Koha/REST/V1/Holding.pm

```diff
@@ -88,6 +88,38 @@ sub get {
 
 sub add {
     my $c = shift->openapi->valid_input or return;
+
+    return try {
+        my $body = $c->validation->param('body');
+        my $biblio = Koha::Biblios->find( $body->{biblionumber} );
+        unless ( $biblio ) {
+            return $c->render( status  => 400,
+                openapi => { error => "Biblio with given biblionumber not found" } );
```

Comment:
This might actually be a "404 Not Found" -error

Reply:
Good point. Did some reading on it and the common opinion was to use 404. Fixed.